### ISSUE TITLE
Move gotosocial from a vps to another

### ISSUE THREAD

**Issue by huabin** [superseriousbusiness/gotosocial#2197]
Hello guys,

I need to move gotosocial to another vps, and I run gotosocial via Docker compose like following

```
version: "3.3"

services:
  gotosocial:
    image: superseriousbusiness/gotosocial:latest
    container_name: gomso
    user: root:root
    networks:
      - gotosocial
    environment:
      GTS_HOST: gomso.com
      GTS_DB_TYPE: sqlite
      GTS_DB_ADDRESS: /gotosocial/storage/sqlite.db
      GTS_LETSENCRYPT_ENABLED: "false"
      GTS_LETSENCRYPT_EMAIL_ADDRESS: ""
    ports:
      - "8080:8080"
    volumes:
      - ~/gomso/data:/gotosocial/storage
      - /etc/timezone:/etc/timezone:ro
      - /etc/localtime:/etc/localtime:ro
    restart: "always"

networks:
  gotosocial:
    ipam:
      driver: default
```

I didn't find `sqlite.db` and media files in local folder `~/gomso/data`. Any ideas? Thank you so much.

**Comment by huabin** [comment]
BTW, the way I'm currently thinking of handling this is to download the data from docker container on the old vps and import it into the docker container on the new vps.

**Comment by tsmethurst** [comment]
Heya! This should indeed be possible just by moving the relevant files and sqlite.* files, and recreating the installation on a new machine. I think a couple folks have already pulled this off.

> download the data from docker container on the old vps and import it into the docker container on the new vps

Not sure what you mean by this; apart from the volumes you mount into it (and `/tmp` and volatile memory) the Docker container for GtS is stateless. You should be able to just copy your `~/gomso/data` directory to your new machine with rsync or something like that, and away you go.

> I didn't find sqlite.db and media files in local folder ~/gomso/data. Any ideas?

No clue, sorry! It should all be in there!

**Comment by huabin** [comment]
Hello @tsmethurst , thank you for the reply. 

I was expecting all the data to be saved in `~/gomso/data` after I installed gotosocial via docker-compose.yaml above.
But the fact is, all the data is not in `~/gomso/data`, it's still just stored in the Docker container under `/gotosocial/storage`.

The steps I took to install that container were

1. Created a folder `gomso`, and go into it.
2. Created a `docker-compose.yaml` file with above code.
3. Created a foder `data`
4. Run the command `docker-compose up -d`

That's all, and I meet this situation now.
